addons/mrp/wizard/change_production_qty.py

```diff
@@ -93,7 +93,14 @@ def change_prod_qty(self):
                     wo.duration_expected = (operation.workcenter_id.time_start +
                                  operation.workcenter_id.time_stop +
                                  cycle_number * operation.time_cycle * 100.0 / operation.workcenter_id.time_efficiency)
-                quantity = wo.qty_production - wo.qty_produced
+                production_qty = wo.qty_production
```

Comment:
Wouldn't this be ok?
```suggestion
                production_qty = wo._get_real_uom_qty(wo.qty_production)
```